src/apps/FlappyBird.jsx

```diff
@@ -0,0 +1,362 @@
+import { useEffect } from "react";
+
+//=========================================
+
+const styles = {};
+styles.stage = {
+  display: "block",
+  maxWidth: "90vw",
+  borderRadius: "10px",
+  boxShadow: "0 0 20px rgb(0, 0, 0, 0.8)",
+};
+
+styles.stageContainer = {
+  display: "block",
+  width: "100%",
+  margin: "20px 0",
+};
```

Comment:
We use styled components for styling our apps, although this isn't wrong by any means, this can be converted into styled components just by making a few changes: 

```suggestion
const Stage = styled.canvas`
  display: block;
  maxWidth: 90vw;
  borderRadius: 10px;
  boxShadow: 0 0 20px rgb(0, 0, 0, 0.8);
`;

const StageContainer = styled.div`
  display: block;
  width: 100%;
  margin: 20px 0;
`;
```

How is this better you make ask? Styled component is used to remove the dependency on obscure classnames but rather to name the components on what they do. You can also notice that we are writing normal css rather than key value pairs for a object.